src/components/Filters.tsx

```diff
@@ -39,52 +69,108 @@ const Filters: React.FC<FiltersProps> = ({
   handleParatimesChange,
   handleMaintainedByOasisToggle,
   handleClearTags,
-  handleClearLangs,
 }) => {
+  const handleTagsChange = (event: SelectChangeEvent<typeof selectedTags>) => {
+    const {
+      target: { value },
+    } = event;
+    const newTags = typeof value === 'string' ? value.split(',') : value;
+    newTags.forEach((tag: string) => handleTagClick(tag));
+  };
```

Comment:
This doesn't work right; and is `split(',')` needed?

https://github.com/user-attachments/assets/c16fb456-81ad-4784-8ab2-953781fc933a

Reply:
Fixed.